webrecorder/browsertrix
# New org home page dashboard

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1188



### Changes

- New org overview page view
- Updates org home page to new view
- Refactors browser profile "new" dialog to use `<btrix-dialog>`

**Differences from mockups**:
- Moved "Total Pages" to "Pages Crawled" since only crawled pages are included
- Added "Archived Items" total (@Shrinks99: Feel free to suggest alternative icon)
- Added "Add New..." dropdown to "Storage". We can move the "Add New" shortcut to the header in this follow up: https://github.com/webrecorder/browsertrix-cloud/pull/1202

### Manual testing

1. Log in. Verify new home view is shown with the correct metrics
2. Click any nav and then "Browsertrix Cloud". Verify you're taken to new home view
3. Complete each "Add New" and "New" flow. Verify new resources are created as expected
4. Regression test Browser Profiles -> "New Browser Profile"
5. Log out and log in as super admin
6. Click org. Verify new home view is shown
7. Switch orgs. Verify you're taken to org's overview
8. Go to an org with quota reached (e.g. "Org with Limits" on dev.) Verify "Add New..." button is disabled

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Home | <img width="1132" alt="Screenshot 2023-09-19 at 4 14 12 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/7aabe4fb-6adb-4f4d-a3bd-469e1ef151fe"> |
| Home - Storage "Add New..." dropdown | <img width="186" alt="Screenshot 2023-09-19 at 4 14 28 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/7accc0be-b473-474d-8cb5-6db0ab79c596"> |
| Home - Adding new browser profile | <img width="1147" alt="Screenshot 2023-09-19 at 4 14 39 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/fec0329a-dbd3-4efa-b311-af4f4a0c2c00"> |
